### ISSUE TITLE
Lost Views (tabs) and Developer tools

### ISSUE THREAD

**Issue by cociweb** [home-assistant/frontend#18757]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

From time to time (inconsistently, maybe when it loads from cache?) I lost temporarily the views (tabs) of my dashboards: Only the first tab appears.
The situation is similar to Developer tools: in this case neither the first tab appears, so there is no tabs ( so none of the yaml, states, services, etc are available)
The situation is the same in the mobile companion app and in desktop browser too.

The only workaround to kill the app! and restart it, or fully re-load the page in the browser (ctrl+F5).

### Describe the behavior you expected

tabs should be always visible.

### Steps to reproduce the issue

Not exactly reproducable (root cause not known).

### What version of Home Assistant Core has the issue?

2023.11

### What was the last working version of Home Assistant Core?

2023.10

### In which browser are you experiencing the issue with?

edge, companion app

### Which operating system are you using to run this browser?

Ubuntu 22.04, Android 13

### State of relevant entities

```txt
-
```

### Problem-relevant frontend configuration

```yaml
-
```

### Javascript errors shown in your browser console/inspector

![image](https://github.com/home-assistant/frontend/assets/1935413/f68674d9-04c8-4cf0-b77f-48beaac1c778)

### Additional information

![Screenshot_20231125-095659.png](https://github.com/home-assistant/frontend/assets/1935413/b0c656cf-12b2-4e35-9b51-9ad528088cd0)

**Comment by karwosts** [comment]
I suggest you try restarting in safe mode, and see if it can still be reproduced in safe mode.

**Comment by cociweb** [comment]
> I suggest you try restarting in safe mode, and see if it can still be reproduced in safe mode. 

Thank you for your suggestion. Fortunatelly? the safe mode solved it. As of now, I'm really out of hints, what component do this behaviour. I've tried   to disable my custom components one by one (without deleting the cache unfortunatelly) but it doesn't popped out where the root cause is.

**Comment by cociweb** [comment]
Hello, 
Just for the record: I've disabled alll the integrations and hacs addons systematically, and it does not solved the problem. Once upon a time I've (re-)discovered the resources page of the lovelace dashboards, where there was a manually added  tv remote editor .js file from a source of the devil's coat. :D - I've added it years ago, and totally forget it. This integration was not used by any cards, but it was loaded....

Cheers,
